plugin/remove-cookie-header.js

```diff
@@ -0,0 +1,16 @@
+module.exports = removeCookieHeader
+removeCookieHeader.attributes = {
+  name: 'removeCookie'
+}
+
+function removeCookieHeader (server, options, next) {
+  server.ext({
+    type: 'onRequest',
+    method: function (request, reply) {
+      // Change all requests to '/test'
```

Comment:
oh sorry, I missed this before. What is this comment about? I think we can remove it?

You can remove the comment, then run `git commit plugin --amend -m 'fix: remove cookie from requests'`, that will update the last commit. Then you can do `git push --force origin master` again